helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -12,7 +25,7 @@ kubeApiServer:
     metricRelabelings:
       - action: keep
         sourceLabels: [__name__]
-        regex: '(apiserver_request_count|apiserver_request_latencies_bucket|apiserver_client_certificate_expiration_seconds_count|apiserver_request_duration_seconds_sum|apiserver_request_duration_seconds_count|APIServiceRegistrationController_depth)'
+        regex: '(process_cpu_seconds_total|process_resident_memory_bytes|go_goroutines|workqueue_queue_duration_seconds_bucket|workqueue_adds_total|workqueue_depth|apiserver_request_total|apiserver_request_count|apiserver_request_latencies_bucket|apiserver_client_certificate_expiration_seconds_count|apiserver_request_duration_seconds_sum|apiserver_request_duration_seconds_count|apiserver_request_duration_seconds_bucket|APIServiceRegistrationController_depth)'
```

Comment:
Have you done any investigation on how much more metrics we get by enabling these? If I remember correctly we disabled a few of these because they produced a very large amount of metrics, if that is still true then we should maybe consider if it is worth to add them back.

Reply:
from what I can see in my dev cluster apiserver_request_duration_seconds_bucket produces the most metrics.
here are some numbers for 1h: `sc -> ~13946, wc -> ~15010`

by removing this metrics the following will not work:
- Prometheus rules: all apiserver_request:burnrate and cluster_quantile:apiserver_request_duration_seconds:histogram_quantile
- Grafana dashboard: Kubernetes / API server
- Prometheus alerts: KubeAPIErrorBudgetBurn

I added some screenshots in the PR description.

Reply:
If I understand the numbers [here](https://github.com/elastisys/ck8s-ops/issues/852) we currently have ~ 75 000-100 000 time series per cluster right now. So increasing it with ~15 000 is big, but not extreme. I'm ok with testing it and maybe reverting it later if we feel the need.